### PR TITLE
[stable/prometheus-operator] Upgrading prometheus version to v2.5.0

### PR DESCRIPTION
**Pull request by harshal-shah**
- Upgrading prometheus-operator version to 0.26.0
- Upgrading prometheus version to 2.5.0
- Upgrading alertmanager version to 0.15.3
- Tested on GKE by installing prometheus operator with above versions.
- helm lint passed.

Signed-off-by: Harshal Shah <harshal2623@gmail.com>

#### What this PR does / why we need it:
prometheus operator released a new version. This PR uses the new version of this operator. 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by paskal** [comment]
Could you please alter your changes so they won't interfere with https://github.com/helm/charts/pull/9647 and most important do not update prometheus-operator to 0.26.0 because of https://github.com/coreos/prometheus-operator/issues/2182?

**Comment by harshal-shah** [comment]
I tested quickly with persistent storage and ran into coreos/prometheus-operator#2182

#9647 is doing pretty much the same thing. I will close this and keep test #9647 on GKE whenever it is ready

**Comment by harshal-shah** [comment]
closing in favor of #9647

**Comment by paskal** [comment]
@harshal-shah, I think updating prometheus and alertmanager is worth to do before updating prometheus-operator, as well as adding `stable/prometheus-operator/CONTRIBUTING.md`. Could you please reopen this PR with only these changes? Thank you for your work!

**Comment by harshal-shah** [comment]
@paskal Added a new commit and reopened the PR.

File: stable/prometheus-operator/Chart.yaml [hunk 1]
```diff
@@ -9,8 +9,8 @@ name: prometheus-operator
 sources:
   - https://github.com/coreos/prometheus-operator
   - https://coreos.com/operators/prometheus
-version: 0.1.29
-appVersion: "0.25.0"
+version: 0.1.30
+appVersion: "0.26.0"
```
Comment (paskal) [inline stable/prometheus-operator/Chart.yaml]: should be 0.25.0

**Review by paskal — CHANGES_REQUESTED** [review]

**Review by paskal — APPROVED** [review]

**Pull request merged**